frontend/src/scenes/userLogic.ts

```diff
@@ -250,6 +250,14 @@ export const userLogic = kea<userLogicType>([
                 return user?.theme_mode || 'light'
             },
         ],
+
+        isUserNonTechnical: [
+            (s) => [s.user],
+            (user) => {
+                const technicalRoles = ['engineering', 'founder']
```

Comment:
I'd argue `data` and `product` are technical so it's just `sales`, `marketing`, `leadership` and `other` that are not.

Reply:
Given we want to keep the experience similar for most users and roll this out to the most non technical users I'd argue we don't need to follow the ICP score exactly. We can also expand it over time but it may be better to limit the exposure of this feature to start.